themes/doks/layouts/shortcodes/glossary.html

```diff
@@ -3,13 +3,15 @@
 {{/*The "more" field in glossaryitems.json displays any additional content displayed by expanding the paragraph on the Glossary page.*/}}
 
 {{ $glossarylist := getJSON "content/en/platform/corda/5.0/glossaryitems.json" }}
+{{ if eq .Page.Params.version "Corda 5.1" }}
+    {{ $glossarylist := getJSON "content/en/platform/corda/5.1/glossaryitems.json" }}
+{{ end }}
```

Comment:
please consider a switch case or if ... elseif...else (which will error) 
Very quickly we will get 5.0, 5.1,5.2, etc